Makefile

```diff
@@ -66,14 +67,24 @@ misspell:	$(MISSPELL)
 misspell-correction:	$(MISSPELL)
 	$(MISSPELL) -w $(ALL_DOCS)
 
+.PHONY: normalized-link-check
+normalized-link-check:
+	@if grep -R '\.\./docs/' docs model/database; then \
+		echo "\nERROR: Found occurrences of '../docs/'; see above."; \
```

Comment:
BTW, should we just enforce absolute paths everywhere? relative is always error prone

Reply:
I think that relative paths are useful. Besides, I might be able to eliminate the need for this normalization, but I'll experiment with that later, once I'm done with https://github.com/open-telemetry/opentelemetry.io/issues/6101.